libs/application/templates/general-petition/src/fields/ListSubmitted/index.tsx

```diff
@@ -16,12 +16,12 @@ const ListSubmited: FC<FieldBaseProps> = ({ application }) => {
   const isStaging = window.location.origin.includes('beta.staging01.devland.is')
 
   const baseUrlForm = isLocalhost
-    ? 'http://localhost:4200/undirskriftalistar/'
```

Comment:
wouldn't it be better to just do
```
baseUrlFrom = `${document.location.origin}/medmaelendalistar/`
```
and skip all the isStaging / isDev business?

Reply:
ok vá! takk!